src/sagemaker/local/local_session.py

```diff
@@ -143,6 +143,10 @@ def create_endpoint(self, EndpointName, EndpointConfigName):
         LocalSagemakerClient._endpoints[EndpointName] = endpoint
         endpoint.serve()
 
+    def update_endpoint(self, EndpointName=None, EndpointConfigName=None):
+        raise Exception('Update endpoint name is not supported in local session.'
```

Comment:
'Update endpoint is not supported in local session.' Should suffice.

You can make recommendation to tell the user to restart their endpoint locally, if they want to change it.